### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by IkariDevGIT** [oobabooga/text-generation-webui#2231]
### Describe the bug

i load a model with offload layer or whatever and it just crashes, nothing.. I've been trying to get this to work for ages! idk what to call this issue btw sorry for the hassle.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

i dont know anymore man

### Screenshot

![grafik](https://github.com/oobabooga/text-generation-webui/assets/90197782/4914ce63-0615-47b8-9a6a-2047947a229d)
![grafik](https://github.com/oobabooga/text-generation-webui/assets/90197782/94f19895-4f57-4e4c-84c0-771bbd119e1a)

### Logs

```shell
no errors
```

### System Info

```shell
windows 10
not using colab
rtx 3070 (msi) 8gvram
```

**Comment by Murtazaraza2002** [comment]
![image](https://github.com/oobabooga/text-generation-webui/assets/23151214/de2805c5-3906-4c3e-8dd8-67edb4b5e10c)
![image](https://github.com/oobabooga/text-generation-webui/assets/23151214/e22f2aa6-a08d-4899-9990-cc79b5d29aca)
Exact same issue.
Others models are working perfectly fine. It says Done! in cmd even though it keep showing loading when I try to load the 13B model or switch from 6B to this 13B model 
![image](https://github.com/oobabooga/text-generation-webui/assets/23151214/e189d304-6199-4983-80bb-7547df6e042e)

**Comment by jllllll** [comment]
I don't think that notstoic model is compatible with the version of GPTQ that this webui uses.
